lib/src/wiggle_abi/req_impl.rs

```diff
@@ -458,4 +458,11 @@ impl FastlyHttpReq for Session {
 
         Ok(outcome)
     }
+
+    fn close(&mut self, req_handle: RequestHandle) -> Result<(), Error> {
+        //·We·don't·do·anything·with·the·parts,·but·we·do·pass·the·error·up·if
+        //·the·handle·given·doesn't·exist
```

Comment:
i think editor whitespace markers got copied in here?

Reply:
They did!